test/module/shared_model/builders/protobuf/test_transaction_builder.hpp

```diff
@@ -18,22 +18,41 @@
 #ifndef IROHA_TEST_TRANSACTION_BUILDER_HPP
 #define IROHA_TEST_TRANSACTION_BUILDER_HPP
 
+#include <memory>
+
 #include "builders/protobuf/builder_templates/transaction_template.hpp"
 #include "module/shared_model/validators/validators.hpp"
 
+using ProtoTxType = shared_model::proto::Transaction;
+
 /**
  * Builder alias, to build shared model proto transaction object avoiding
  * validation and "required fields" check
  */
 using TestTransactionBuilder = shared_model::proto::TemplateTransactionBuilder<
     (1 << shared_model::proto::TemplateTransactionBuilder<>::total) - 1,
     shared_model::validation::AlwaysValidValidator,
-    shared_model::proto::Transaction>;
+    ProtoTxType>;
 
+/**
+ * Builder for creating \class shared_model::proto::UnsignedWrapper of \class
+ * ProtoTxType
+ */
 using TestUnsignedTransactionBuilder =
     shared_model::proto::TemplateTransactionBuilder<
         (1 << shared_model::proto::TemplateTransactionBuilder<>::total) - 1,
         shared_model::validation::AlwaysValidValidator,
-        shared_model::proto::UnsignedWrapper<shared_model::proto::Transaction>>;
+        shared_model::proto::UnsignedWrapper<ProtoTxType>>;
+
+/**
+ * Wrapper for making shared_ptr from the passed builder
```

Comment:
should be written: .. for making shared_ptr on transaction from the passed builder